locales/pt.json

```diff
@@ -1,215 +1,215 @@
+	"loading solc compiler": "carregando o compilador solc",
+	"Welcome to Embark": "Bem-vindo ao Embark",
+	"possible commands are:": "comandos possíveis são:",
+	"display versions in use for libraries and tools like web3 and solc": "lista versões em uso para bibliotecas e ferramentas como web3 e solc",
+	"instantiated web3.js object configured to the current environment": "objeto web3.js instanciado configurado para o ambiente atual",
+	"to immediatly exit (alias: exit)": "para sair imediatamente (atalho: exit)",
+	"The web3 object and the interfaces for the deployed contracts and their methods are also available": "O objeto web3 e as interfaces para os contratos implantados e seus métodos também estão disponíveis",
+	"create a working dapp with a SimpleStorage contract": "Cria uma dapp funcional com o contrato SimpleStorage",
+	"filename to output logs (default: none)": "ficheiro/arquivo para saída dos logs (predefinido: none)",
+	"level of logging to display": "nivel do log",
+	"deploy and build dapp at ": "Publica os contractos e constroi a applicacao em ",
+	"port to run the dev webserver (default: %s)": "porta para correr o servidor wen para desenvolvimento (default: %s)",
+	"host to run the dev webserver (default: %s)": "host para correr o servidor wen para desenvolvimento (default: %s)",
+	"disable the development webserver": "desativa o servidor wen para desenvolvimento",
+	"simple mode, disables the dashboard": "modo simples, disativa o dashboard",
+	"no colors in case it's needed for compatbility purposes": "sem cores, em caso seja necessario para compabitilidade com a terminal",
+	"filename to output logs (default: %s)": "ficheiro/arquivo para os logs (predefinido: %s)",
+	"run dapp (default: %s)": "executa a dapp (applicacao decentralizada) (predefinido: %s)",
+	"Use a specific ethereum client or simulator (supported: %s)": "Usa um cliente ou simulador de ethereum específico (supportado: %s)",
+	"run blockchain server (default: %s)": "executa un node de blockchain (predefinido: %s)",
+	"run a fast ethereum rpc simulator": "executa um simulador RPC de ethereum",
+	"use testrpc as the rpc simulator [%s]": "usa testrpc como simulator de rpc [%s]",
+	"port to run the rpc simulator (default: %s)": "porta para executar simulador de rpc (predefinido: %s)",
+	"host to run the rpc simulator (default: %s)": "host para executar servidor de rpc (predefinido: %s)",
+	"number of accounts (default: %s)": "número de contas (predefinido: %s)",
+	"Amount of ether to assign each test account (default: %s)": "Quantidade de ether para atribuir cada conta de teste (predefinido: %s)",
+	"custom gas limit (default: %s)": "limite de gás (predefinido: %s)",
+	"run tests": "executar os testes",
+	"resets embarks state on this dapp including clearing cache": "redefinindo o estado do Embark nesta appliacao, incluindo a cache",
```

Comment:
`redefinindo` is still not the right translation here. We're not changing state, we're cleaning up.